rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -896,14 +897,19 @@ def test_maybe_create_calendar_reminder(
         assert calendar_db.query_calendar_entry(CalendarFilterQuery.make())['entries_total'] == 0
         return  # finish this test case because this setting will prevent creation of reminders
 
+    latest_expiry_of_ens: dict[str, Timestamp] = defaultdict(lambda: Timestamp(0))
+    for event in ens_events:
+        if event.extra_data is not None and latest_expiry_of_ens[event.extra_data['name']] < event.extra_data['expires']:  # noqa: E501
+            latest_expiry_of_ens[event.extra_data['name']] = event.extra_data['expires']
```

Comment:
this is something that you know beforehand when creating the test right? Instead of creating dinamically the test create it manually. Sometimes there are bugs that are not spotted because you expect data that you don't have